cmd/sfncli/runner.go

```diff
@@ -76,20 +85,18 @@ func (t TaskRunner) Process(ctx context.Context) error {
 
 	// AWS requires JSON output. If it isn't, then make it so.
 	output := stdoutbuf.String()
-	var test interface{}
-	if err := json.Unmarshal([]byte(output), &test); err != nil {
-		// output isn't JSON, make it json and stay under the length limit
-		if len(output)+100 > maxTaskOutputLength && len(output) > 100 {
-			output = output[100:] // stay under the limit
-		}
-		newOutputBs, _ := json.Marshal(map[string]interface{}{
-			"raw": output,
-		})
-		output = string(newOutputBs)
+	var out map[string]interface{}
+	if err := json.Unmarshal([]byte(output), &out); err != nil {
+		return fmt.Errorf("Worker must output json object to stdout: %s", err)
```

Comment:
have you verified that this error message gets back to AWS (i.e. it get's put into the `SendTaskFailure` call)? Want to make sure users have good feedback on this if we're going to be strict

Reply:
Good catch.  Tasks weren't being marked as failures if the input was invalid.  Fixed in 
262f7b9